src/app/compare/compare.html

```diff
@@ -19,7 +19,7 @@
         <span ng-bind="item.primStat.value"></span>
       </div>
       <div ng-class="{ highlight: vm.highlight === stat.statHash }" ng-mouseover="vm.highlight = stat.statHash" ng-repeat="stat in item.stats track by $index" ng-style="stat | statRange:vm.statRanges | qualityColor:'color'">
-        <span ng-bind="::stat.value"></span>
+        <span ng-bind="stat.value || 'N/A'"></span>
```

Comment:
This should not be a hard-coded string as 'N/A' does not mean the same thing in all locales, thus this should be added via i18n

Reply:
Updated although I'm honestly not super familiar with i18n in angular so if there is something else you had in mind just let me know.